x-pack/platform/plugins/private/file_upload/server/plugin.ts

```diff
@@ -32,6 +38,40 @@ export class FileUploadPlugin implements Plugin {
 
     setupCapabilities(coreSetup);
 
+    plugins.features.registerKibanaFeature({
```

Comment:
Is there a way to give it a default value of 'All'? Otherwise for existing custom roles, it is showing as 'None' so any user who had a single custom role before this change will have no file upload access where before they did have access.